src/main/twirl/uk/gov/hmrc/play/views/layouts/head.scala.html

```diff
@@ -28,6 +29,10 @@
 	<link rel='stylesheet' href='@{AssetsConfig.assetsPrefix}/stylesheets/application.min.css' />
 	<!--<![endif]-->
 
+	@for(id <- optimizelyProjectId){
```

Comment:
Yes, this works similar to map but it won't render anything if the `Option` value is ` None` and you also don't need a `getOrElse`